index.html

```diff
@@ -75,136 +78,219 @@ <h3 class="title">My Recent Works</h3>
+        </p>
+      </div>
+      <div><button type="button" class="about-me-btn">Get my resume</button></div>
       <img class="right-side-image-b" src="images/illustration-about-me-mobile.png" alt="right-design about-me section">
       <img class="hor-line" src="images/Devider.png" alt="horizental line">
       <div class="list-container">
         <ul class="about-me-cat">
           <li class="language">
             Languages
-              <div class="category-list-container">
-                <div class="category-list">Javascript</div>
-                <div class="category-list">Ruby</div>
-                <div class="category-list">CSS</div>
-                <div class="category-list">HTML</div>
-              </div>
+            <div class="category-list-container">
+              <div class="category-list">Javascript</div>
+              <div class="category-list">Ruby</div>
+              <div class="category-list">CSS</div>
+              <div class="category-list">HTML</div>
+            </div>
           </li>
           <li class="frameworks">
             Frameworks
-              <div class="category-list-container">
-                <div class="category-list">React.js</div>
-                <div class="category-list">Ruby on Rails</div>
-                <div class="category-list">RSpec</div>
-                <div class="category-list">Capybara</div>
-              </div>
+            <div class="category-list-container">
+              <div class="category-list">React.js</div>
+              <div class="category-list">Ruby on Rails</div>
+              <div class="category-list">RSpec</div>
+              <div class="category-list">Capybara</div>
+            </div>
           </li>
           <li class="skills">
             Skills
-              <div class="category-list-container">
-                <div class="category-list">Database Management</div>
-                <div class="category-list">Version Control</div>
-                <div class="category-list">CLI</div>
-                <div class="category-list">API Design</div>
-                <div class="category-list">Web Development</div>
-              </div>
+            <div class="category-list-container">
+              <div class="category-list">Database Management</div>
+              <div class="category-list">Version Control</div>
+              <div class="category-list">CLI</div>
+              <div class="category-list">API Design</div>
+              <div class="category-list">Web Development</div>
+            </div>
           </li>
         </ul>
       </div>
     </section>
+
+
+    <!--  contact form section -->
+    <!-- added by habtamualemayehu16@gmail.com   -->
+    <section class="reg-form">
+      <div class="form-body">
+        <div class="header-text">
+          <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
```

Comment:
- Great job so far @forLoop94 your form works great. There are just a few style adjustments you need to make. You can see from the image below that your contact session is missing some padding as compared to the Figma design. There isn't enough spacing between the `Skills section` and the `Contact` section so please fix this so your design looks like the Figma design :100: 
- [ ]  You also need to increase the width of the contact page header text. It has a smaller width compared to the Figma Design :100: 

| Your Design | The Figma Design |
|--------|--------|
| ![Screenshot from 2023-03-29 15-18-17](https://user-images.githubusercontent.com/102699958/228591673-1bd3aba3-cf95-403b-9042-cb424a4512d8.png) | ![Screenshot from 2023-03-29 15-24-37](https://user-images.githubusercontent.com/102699958/228591249-c4c1ed4d-bd29-476a-8821-c564e6bfb086.png) |